package.json

```diff
@@ -20,6 +21,13 @@
     "typescript": "4.9.5"
   },
   "devDependencies": {
-    "@nypl/design-system-react-components": "2.0.1"
+    "@nypl/design-system-react-components": "2.0.1",
+    "@testing-library/jest-dom": "^6.1.4",
```

Comment:
Let's pin the specific versions for now. Sometimes, minor update or patches can install new packages or possibly break something. This means removing the caret `^` for all these packages.